src/angular/9-form-value-changes/observables-subscriptions.html

```diff
@@ -74,7 +74,7 @@
     `
 })
 class DestroyComponent implements OnInit, OnDestroy {
-  $mySubscription: Subscription;
+  mySubscription: Subscription;
```

Comment:
I like that we're adopting the `unSubscribe` pattern for handling subscriptions on `ngOnDestroy`. Is it within scope of this refactor to update here as well?